### DISCUSSION TITLE
Addons language

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#2345] [Help] [answered]
How do I change the language of the Scratch Addons settings page? I tried changing the language in the Scratch website, Scratch editor, browser (chrome) but I keep seeing the addons settings page in the wrong language.

**Comment by cobaltt7** [comment]
It should sync with your Scratch language unless the language you choose is not translated.

What language are you trying to use?

**Reply by mxmou** [reply]
Only addons that add features to Scratch use the Scratch language. The settings page uses the browser language.

**Comment by mxmou** [comment, marked as answer]
The settings page uses the language of the browser.

**Reply by WorldLanguages** [reply]
The language of the _browser UI_ - the language that after changing, you need to restart your browser

**Comment by sfederici** [comment]
I changed the language of the chrome browser in settings > advanced > languages to English and restarted chrome. The addons settings were still in Italian. Then I changed the language of Google to English, following directions at https://www.businessinsider.com/how-to-change-language-in-google?IR=T and restarted chrome. The addons settings are still in Italian.

**Reply by WorldLanguages** [reply]
Please go to the Scratch Addons settings page, open devtools console and paste this:
`copy({1: navigator.language, 2:navigator.languages, 3: chrome.i18n.getUILanguage()})`
The result will be copied to your clipboard. Paste here so that I can see it

**Reply by sfederici** [reply]
I got
{
    "1": "en-US",
    "2": [
        "en-US",
        "it-IT",
        "it",
        "en",
        "fr"
    ],
    "3": "it"
}

**Reply by WorldLanguages** [reply]
`chrome.i18n.getUILanguage()` is returning `"it"`, are you sure your browser is set to English? The browser UI must be set to English (for example, opening an incognito window should show the incognito warning/info in English, not Italian)

**Reply by mxmou** [reply]
In the language settings, click the menu next to Italian and enable "Display Google Chrome in this language".

**Reply by sfederici** [reply]
I went to chrome > settings > advanced > languages, I opened the language menu. If I click the three-button menu next to a language I see the following options: 1) propose translation of this language, 2) move up, 3) move down, 4) remove. There is no "Display Google Chrome in this language" option

**Reply by WorldLanguages** [reply]
Well we can't help you with that. This is a Chrome issue, it's not letting you change the browser UI language. You need to find a way to do that

**Reply by sfederici** [reply]
I discovered that, incredibly, in macOS to change the browser's language you have to change your system language. So, I guess I will try to translate the Italian names of my addons back to English. I'm sorry if this will make difficult for you to quickly identify which addon I'm referring to.

**Reply by WorldLanguages** [reply]
You could try using the search function on this GitHub repository and find the file inside /addons-l10n/it that has the translated addon name. The name of the file (minus the .json) is the addon ID. You can then simply use the addon ID or find the English name on /addons/[addon-id]/addon.json

**Comment by sfederici** [comment]
BTW, what I would like to see is the English name of each addon, so that I can refer to the exact name in the Discussions.

**Reply by cobaltt7** [reply]
You can also Ctrl+Click on any addon name to see the addon IDs used internally. Those are always the same, regardless of language.

**Reply by mxmou** [reply]
@RedGuy12 This feature is only available in prereleases, not in the stable version. Addon IDs can be found [here](https://scratchaddons.com/addons).

**Reply by sfederici** [reply]
Even if it is not very handy, I think I can refer to the page https://scratchaddons.com/addons
